c2corg_ui/templates/outing/helpers/detailed_outings_attributes.html

```diff
@@ -160,3 +170,36 @@
     </div>
   %endif
 </%def>
+
+## CONDITIONS LEVELS
+<%def name="get_outing_conditions_levels(locale)">\
+  %if locale.get('conditions_levels'):
+      <div class="view-details-info col-xs-12" ng-init="conditions = ${locale['conditions_levels']}">
+        <h3 class="heading closed show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" data-toggle="collapse" data-target="#conditions_levels">
+          <span translate>conditions_levels</span><span class="glyphicon glyphicon-menu-right"></span>
+        </h3>
+        <section class="collapse" id="conditions_levels">
+          <div class="lead table-responsive">
+            <table class="table table-striped conditions-levels">
+              <thead>
+                <tr>
+                  <th class="location" translate>location / altitude / orientation / time</th>
```

Comment:
I think using separated translations here would help reducing the number of strings to translate:
```
<th class="location"><span translate>location</span> / <span translate>elevation</span> / <span translate>orientation</span> / <span translate>time</span></th>
```